### PR TITLE
fix: fw-775 introduce id to Post

### PR DESCRIPTION
**Pull request by swkatmask**
https://mask.atlassian.net/browse/fw-775

### CODE REVIEW

File: src/providers/types/SocialMedia.ts [hunk 1]
```diff
@@ -89,6 +89,12 @@ export interface Attachment {
 export type PostType = 'Post' | 'Comment' | 'Quote' | 'Mirror';
 
 export interface Post {
+    /**
+     * For Farcaster, it's hash of the cast.
+     * For Lens, it's id of the publication, which is different from post id.
+     * TODO id for Twitter
+     */
+    id: string;
```
Comment (guanbinrui) [inline src/providers/types/SocialMedia.ts:97]:
```suggestion
    publishedPostId: string;
```

Comment (swkatmask) [reply src/providers/types/SocialMedia.ts:97]: Actually, it's not **post** id, it's **publication** id. A publication could be a mirror action or some others.

Comment (guanbinrui) [reply src/providers/types/SocialMedia.ts:97]: Okay, what about `publicationId`?

File: src/providers/types/SocialMedia.ts [hunk 2]
```diff
@@ -89,6 +89,12 @@ export interface Attachment {
 export type PostType = 'Post' | 'Comment' | 'Quote' | 'Mirror';
 
 export interface Post {
+    /**
+     * For Farcaster, it's hash of the cast.
+     * For Lens, it's id of the publication, which is different from post id.
+     * TODO id for Twitter
+     */
+    publicationId: string;
```
Comment (guanbinrui) [inline src/providers/types/SocialMedia.ts:97]:
```suggestion
    publicationId?: string;
```

**Pull request merged**